install_script.sh

```diff
@@ -0,0 +1,49 @@
+#!/usr/bin/env bash
+
+# References
+# http://kvz.io/blog/2013/11/21/bash-best-practices/
+# http://jvns.ca/blog/2017/03/26/bash-quirks/
+
+# exit when a command fails
+set -o errexit
+
+# exit if any pipe commands fail
+set -o pipefail
+
+set -E
+set -o functrace
+function handle_error {
+    local retval=$?
+    local line=${last_lineno:-$1}
+    echo "Failed at $line: $BASH_COMMAND"
+    echo "Trace: " "$@"
+    echo "return code: " "$?"
+    exit $retval
+ }
+trap 'handle_error $LINENO ${BASH_LINENO[@]}' ERR
+
+CONDA_EXE="$(which conda)"
```

Comment:
@William-Hill does this work if conda is a function rather than a actual executable (remember issue you had where `cdat_info` couldn't find your `conda` executable for provenance?